### DISCUSSION TITLE
Bromite doesn't Block Ads On Google Searches

### DISCUSSION THREAD

**Discussion by mfc1337x** [bromite/bromite#1240] [Q&A]
When I search for a product on Google, bromite shows me ads even though I have adblocking enabled.

**App Version:**
Latest Version from F-Droid Client

**Android Version:**
11

**Here are screenshots of my issue:**
![Screenshot_20210628-001147__01](https://user-images.githubusercontent.com/59104299/123561158-237f7d00-d7a7-11eb-9159-58dde4e3fddc.jpg)
![Screenshot_20210628-001323__01](https://user-images.githubusercontent.com/59104299/123561162-25494080-d7a7-11eb-88cd-78cb0b5fea54.jpg)

**What can I do to fix this behaviour?**

**Comment by csagan5** [comment]
Duplicate of #1143
